source/_patterns/02-molecules/interactive/dropdown/dropdown.js

```diff
@@ -1,6 +1,6 @@
 import easydropdown from "easydropdown";
 
-const selectElements = document.querySelectorAll("select:not([multiple])");
+const selectElements = document.querySelectorAll(".jcc-dropdown select:not([multiple])");
 
```

Comment:
@slowbot Does this mean we don’t want to style select elements using EDD by default? AFAIK, the small size was going to be the global default, with the bigger one being used in other select areas. This was what we implemented in Drupal, so curious what the intention is, and If we’ll need to change it.  If you could please let me know, I’d appreciate it. Thanks!